README.md

```diff
@@ -959,26 +959,6 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
```

Comment:
> This is a really old rule (that was [added](https://github.com/airbnb/swift/commit/c93ee5aae718d25401e001e78308bef73a4b5ac9#diff-9ba7d4d50c31dd7166688703e242142dR9) in 2016!). I don't think this rule represents idiomatic Swift anymore, now that the language has evolved and `enum` namespaces have become much more common.

What new information has come to light between now and 2016 that indicates that this is pattern is no idiomatic to Swift?

Reply:
My general feeling is that the community has moved away from using unnamespaced globals over the past few years (which, anecdotally, seemed a lot more common in 2016 than it seems today in 2020).

Reply:
That may be true though I feel like it's hard to objectively quantify this.